py/client/pydeephaven/_table_interface.py

```diff
@@ -44,8 +44,8 @@ def drop_columns(self, cols: Union[str, List[str]]):
         table_op = DropColumnsOp(column_names=to_list(cols))
         return self.table_op_handler(table_op)
 
-    def update(self, formulas: Union[str, List[str]]):
-        """ Perform an update operation on the table and return the result table.
+    def update(self, formulas: Union[str, List[str]]) -> Any:
+        """The update method creates a new table containing a new, in-memory column for each formula.
```

Comment:
I'm confused by the typehints and docs in this file.  The docs say that tables are returned, but all of the return types are marked as `Any`.